homeassistant/components/homewizard/manifest.json

```
@@ -7,6 +7,7 @@
   "requirements": ["python-homewizard-energy==1.1.0"],
   "zeroconf": ["_hwenergy._tcp.local."],
   "config_flow": true,
+  "quality_scale": "platinum",
```

```suggestion
  "quality_scale": "silver",
```

If this is preferred

---

You don't need to go through all levels. So, this isnt' needed.